e2e/components/drafts/ActionBar.test.ts

```diff
@@ -0,0 +1,37 @@
+import {test, expect} from '@playwright/test'
+import {visit} from '../../test-helpers/storybook'
+import {themes} from '../../test-helpers/themes'
+
+test.describe('ActionBar', () => {
+  test.describe('CommentBox', () => {
+    for (const theme of themes) {
+      test.describe(theme, () => {
+        test('default @vrt', async ({page}) => {
+          await visit(page, {
+            id: 'drafts-components-actionbar--comment-box',
+            globals: {
+              colorScheme: theme,
+            },
+          })
+
+          // Default state
+          await page.getByText('Write').click()
```

Comment:
Do we need to press the "Write" tab for the default state? We might want to take the screenshot of the default state first without performing any action.

Reply:
Oops my bad! we don't need to click it!

Reply:
No worries, let me know when the snapshots are generated, I'll give another review 😊